packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -528,6 +624,56 @@ function makeAugmentedSchema(
             },
         });
 
+        composer.createObjectTC({
+            name: `${node.name}AggregateSelection`,
```

Comment:
There is an inconsistency here, the scalars are "AggregationSelection" and object types are "AggregateSelection" - I think they should all be either one or the other. I believe "AggregateSelection" would be accurate.

Reply:
Yes!